pkg/apis/controlplane/types.go

```diff
@@ -340,6 +340,16 @@ type NodeStatsSummary struct {
 	AntreaClusterNetworkPolicies []NetworkPolicyStats
 	// The TrafficStats of Antrea NetworkPolicies collected from the Node.
 	AntreaNetworkPolicies []NetworkPolicyStats
+	// The stats related to multicast collected from the Node.
+	Multicast []MulticastStats
```

Comment:
For me stats implies things like counters. Right now it seems that we only provide group membership information. Is that going to change in the future? If not, this should probably be renamed. @jianjuns @tnqn

Reply:
If I understand it correctly, you were implying MulticastStats should not be put in NodeStatsSummary API because it doesn't include real stats(only group-pod membership) for now. It should be added to another controlplane API. 

For me creating a new API may seem like overkill. Also, We may add extra stats-like fields in this struct in the future.

Reply:
I think using the same API (`NodeStatsSummary`) is good, even though in a future version maybe the API should be renamed to something more generic (e.g., `NodeSummary`). But yes, you don't have any stats in `MulticastStats` at the moment, and `MulticastGroupInfo` sounds like a more appropriate name.

Reply:
Updated. Thanks for the explanation.